_install-and-configure/configuring-opensearch/index-settings.md

```diff
@@ -143,6 +143,8 @@ OpenSearch supports the following dynamic index-level index settings:
 
 - `index.search.idle.after` (Time unit): The amount of time a shard should wait for a search or get request until it goes idle. Default is `30s`.
 
+- `index.search.default_pipeline` (String): The default search pipeline for the index to be used on each search request. If the default pipeline is set and the pipeline does not exist, then index requests fail. The pipeline name _none specifies that the index does not have an search pipeline.
```

Comment:
```suggestion
- `index.search.default_pipeline` (String): The name of the search pipeline that will be used if no pipeline is explicitly set when searching an index. If a default pipeline is set and the pipeline doesn't exist, then the index requests fail. Use the pipeline name `_none` to specify no default search pipeline. For more information about default search pipelines, see https://opensearch.org/docs/latest/search-plugins/search-pipelines/using-search-pipeline/#default-search-pipeline.
```

Reply:
That looks perfect to me. Thanks a lot!

Reply:
"For more information, see [Default search pipeline](https://opensearch.org/docs/latest/search-plugins/search-pipelines/using-search-pipeline/#default-search-pipeline)." Otherwise, LGTM.